packages/parmap/parmap.1.0-rc1/opam

```diff
@@ -26,4 +26,8 @@ depends: [
   "camlp4"
   "ocamlbuild" {build}
   "conf-autoconf"
+  ]
+depexts: [
```

Comment:
Thanks Anil, will include this fix in the next update

Roberto Di Cosmo (via mobile/cell)
Le 27 avr. 2016 13:23, "Anil Madhavapeddy" <notifications@github.com> a
écrit :

> In packages/parmap/parmap.1.0-rc1/opam
> <https://github.com/ocaml/opam-repository/pull/6324#discussion_r61241528>:
>
> > @@ -26,4 +26,8 @@ depends: [
> >    "camlp4"
> >    "ocamlbuild" {build}
> >    "conf-autoconf"
> > +  ]
> > +depexts: [
>
> since you depend on conf-autoconf, you shouldn't need the explicit
> depexts on autoconf anymore. conf-autoconf also includes support for many
> other platforms so it should be a superset.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ocaml/opam-repository/pull/6324/files/60dd39a271497ba036a297ae42407724aed97d08#r61241528>
>